recipes/nasp/meta.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: nasp
+  version: "1.0.1"
+
+source:
+  fn: nasp-1.0.1.tar.gz
+  url: https://pypi.python.org/packages/source/n/nasp/nasp-1.0.1.tar.gz
+  md5: a6c552062d03ce4cb261164147b9a711
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  skip: True # [py27]
+  # noarch_python: True
+  # preserve_egg_dir: True
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - nasp = nasp:main
+    #
+    # Would create an entry point called nasp that calls nasp.main()
+
+    - nasp = nasp.nasp:main
+    - format_fasta = nasp.format_fasta:main
+    - find_duplicates = nasp.find_duplicates:main
+    - convert_external_genome = nasp.convert_external_genome:main
+    - vcf_to_matrix = nasp.vcf_to_matrix:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  #  number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
```

Comment:
Good catch! Missed to add the modified version including non-python run dependencies. Fixing it but leaving out the optional callers/aligner for now. There are recipes for all optional dependencies except SolSNP so would you prefer to have them also listed?